persistence/pre_persistence/module.go

```diff
@@ -0,0 +1,53 @@
+package pre_persistence
+
+import (
+	"github.com/pokt-network/pocket/shared/config"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/types"
+	"github.com/syndtr/goleveldb/leveldb/comparer"
+	"github.com/syndtr/goleveldb/leveldb/memdb"
+)
+
+func Create(cfg *config.Config) (modules.PersistenceModule, error) {
+	db := memdb.New(comparer.DefaultComparer, 888888888)
```

Comment:
(_Silly question_) Can this come from the config? If it's important at all it should.

Reply:
Yes, will add